src/framework/audio/worker/internal/synthesizers/fluidsynth/fluidsynth.cpp

```diff
@@ -110,6 +108,8 @@ Ret FluidSynth::init()
     fluid_set_log_function(FLUID_INFO, fluid_log_out, nullptr);
     fluid_set_log_function(FLUID_DBG, fluid_log_out, nullptr);
 
+    m_outputSpec = spec;
+
     m_fluid->settings = new_fluid_settings();
     fluid_settings_setnum(m_fluid->settings, "synth.gain", FLUID_GLOBAL_VOLUME_GAIN);
     fluid_settings_setint(m_fluid->settings, "synth.audio-channels", FLUID_AUDIO_CHANNELS_PAIR); // 1 pair of audio channels
```

Comment:
This is [one of such places](https://github.com/musescore/MuseScore/pull/29462#discussion_r2298141022), which can now be easily fixed